### DISCUSSION TITLE
Stadtbücherei Wuppertal

### DISCUSSION THREAD

**Discussion by gigamaz** [stefanw/voebbot#484] [General]
Moin zusammen,

vielleicht sehe ich den Wald vor lauter Bäumen nicht, aber wie richte ich den Zugang zur o.g. Bücherei ein? Wird als unterstützt gelistet, ist im Plugin selbst dann aber keine Option / steht nicht zur Auswahl.

Übersehe ich etwas?

**Comment by Amar0ks** [comment]
Wuppertal ist nicht unterstütz.
https://github.com/stefanw/bibbot/issues/109

**Reply by gigamaz** [reply]
Das steht auf der Projektseite anders, daher meine Verwunderung..
